lib/deploy.js

```diff
@@ -41,9 +44,17 @@ function sftpDeploy(opts) {
         // Wait for the transfer to finish...
         rstdin.once('finish', function() {
           tessel.logs.info('Deployed.');
+          if (push) {
+            tessel.logs.info('Setting up to run on boot...');
+            conn.exec("echo '#!/bin/sh\nnode app' > " + filepath + "/start && chmod +x '" + filepath + "/start'", function (err, newStream) {
+              newStream.on('close', function () {
+                tessel.logs.info('You may now disconnect from the Tessel. Your code will be run whenever Tessel boots up. To remove this code, use `tessel erase`.');
+              });
+            });
+          }
           tessel.logs.info('Running script...');
           // Once it has been written, run the script with Node
-          conn.exec('node /tmp/remote-script/' + ret.relpath, function(err, stream) {
+          conn.exec('node ' + filepath + '/' + ret.relpath, function(err, stream) {
```

Comment:
Start the process with `/etc/init.d/tessel-app/start` so procd tracks it and can kill it later.